src/acceptance-test/java/net/consensys/orion/acceptance/EthClientStub.java

```diff
@@ -182,6 +183,25 @@ ReceiveResponse receivePrivacy(String digest, String publicKey) {
     return Optional.ofNullable(keyFuture.join());
   }
 
+  public Optional<PrivacyGroup> modifyPrivacyGroup(
+      String address,
```

Comment:
add final to params.

Reply:
We're planning to do this all separately, see here: https://pegasys1.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=PAN&modal=detail&selectedIssue=OR-314